server/routes/communityOffenderManager/residentialChecks/task.ts

```diff
@@ -86,9 +86,15 @@ export default class ResidentialChecksTaskRoutes {
       }
     }
 
+    const { user } = res.locals
     const saveAnswersRequest: SaveResidentialChecksTaskAnswersRequest = {
       taskCode,
       answers,
+      agent: {
+        username: user.username,
+        role: 'PROBATION_COM',
+        onBehalfOf: task.assessmentSummary.team,
+      },
```

Comment:
This is fine as is for now - we might want to move it to middleware at some point.